src/index.ts

```diff
@@ -136,9 +136,9 @@ function getTags(client: {
 }
 
 async function getNodeLtsVersions() {
-    const response = await axios.get<NodeVersion[]>('https://nodejs.org/dist/index.json');
+    const response = await axios.get<{ version: string, lts: string|boolean }[]>('https://nodejs.org/dist/index.json');
     const groupedVersion = groupBy(response.data.filter(it => it.lts), 'lts');
-    const versions: { version: string, major: string, lts: string }[] = [];
+    const versions: NodeVersion[] = [];

```

Comment:
?

Reply:
NodeVersion is the same as { version: string, major: string, lts: string } where as on line 139 the this was wrong and https://nodejs.org/dist/index.json never resulted a major value.